### ISSUE TITLE
[CALL_API].credentials is not supported

### ISSUE THREAD

**Issue by tdfairbrother** [agraboso/redux-api-middleware#48]
In the documentation, it talks about [CALL_API].credentials being an acceptable option but when I try and set it to 'same-origin', it fails on the isRSAA check with an error... Error: Actions may not have an undefined "type" property. Have you misspelled a constant?

Fails on the isRSAA check.

```
function apiMiddleware({ getState }) {
  return (next) => (action) => {
    const callAPI = action[CALL_API];
    if (!isRSAA(action)) {
      return next(action);
    }
```

Here is my action...

```
import {CALL_API} from 'redux-api-middleware';

const getUsers = (data) => {
  return {
    [CALL_API]: {
      endpoint: '/api/users',
      method: 'GET',
      credentials: 'same-origin',
      types: ['REQUEST', 'SUCCESS', 'FAILURE']
    }
  };
};

export default {
  getUsers
};
```

Looking through your code I can see no mention of the credentials property. I also see no mention in your tests. Is it because your documentation is wrong or am I doing something wrong?

**Comment by tdfairbrother** [comment]
Ok I worked out that the credentials support is in the beta version which is not the version that installs with npm install.

**Comment by barrystaes** [comment]
@tdfairbrother thanks for the update.
Its since then been released as v1.0.0 on NPM.
